src/DonorProfiles/resources/js/app/tabs/annual-receipts/content.js

```diff
@@ -5,7 +5,7 @@ const Content = () => {
 	return (
 		<Fragment>
 			<Heading>
-				Annual Receipt
+				Annual Receipts
```

Comment:
Does this need to be translatable?

Reply:
This particular string will be addressed in a future PR (when the Annual Receipts tab is actually implemented, currently it is us boilerplate), but I did go through and complete a translation pass on work specifically related to this PR: https://github.com/impress-org/givewp/pull/5455/commits/bad8e4954392ebdfbb0a3530414ac226a4a91b84